terraform/sqs.tf.json.template.py

```diff
@@ -29,18 +33,21 @@
                                 "maxReceiveCount": 9 if retry else 1,
                                 "deadLetterTargetArn": "${aws_sqs_queue.%s.arn}"
                                                        % config.unqual_tallies_queue_name(retry=not retry, fail=retry)
-                            })
+                            }),
+                            "tags": get_cloud_tags(config.tallies_queue_name(retry=retry)),
                         }
                         for retry in (False, True)
                     },
                     config.unqual_notifications_queue_name(fail=True): {
                         "name": config.notifications_queue_name(fail=True),
                         "message_retention_seconds": 14 * 24 * 60 * 60,
+                        "tags": get_cloud_tags(config.qualified_resource_name('fail')),
                     },
                     config.unqual_tallies_queue_name(fail=True): {
                         "fifo_queue": True,
                         "name": config.tallies_queue_name(fail=True),
                         "message_retention_seconds": 14 * 24 * 60 * 60,
+                        "tags": get_cloud_tags(config.qualified_resource_name('fail', suffix='.fifo')),
```

Comment:
And finally we should add a convenience for the most common case:

```suggestion
                    config.unqual_tallies_queue_name(fail=True): add_tags({
                        "fifo_queue": True,
                        "name": config.tallies_queue_name(fail=True),
                        "message_retention_seconds": 14 * 24 * 60 * 60,
```